syft/frameworks/crypten/context.py

```diff
@@ -127,8 +127,7 @@ def wrapper(*args, **kwargs):
 
             rank_to_worker_id = dict(zip(range(0, len(workers)), [worker.id for worker in workers]))
 
-            sy.local_worker.add_crypten_support()
-            sy.local_worker._set_rank_to_worker_id(rank_to_worker_id)
+            sy.local_worker.rank_to_worker_id = rank_to_worker_id
```

Comment:
This is kind of ugly. This ```rank_to_worker_id``` is used in the ```load``` function from ```crypten/__init__py```...Thinking how we could remove this or refactor the ```load``` function

Reply:
We can add some functions that take a worker as argument